eox_tagging/test/test_models.py

```diff
@@ -318,18 +318,49 @@ def test_create_without_default_owner(self):
                 "tag_type": "example_tag_3",
                 "validate_tag_value": {"regex": r".*eduNEXT$"},
                 "validate_target_object": "CourseEnrollment",
-                "validate_expiration_date": {"exist": True, "in": ["Dec 04 2020", "Oct 19 2020"]},
-                "validate_activation_date": "Jun 16 2020"
+                "validate_expiration_date": {"exist": True, "between": ["2020-10-19 10:20:30", "2020-12-04 10:20:30"]},
+                "validate_activation_date": "2020-06-16 10:20:30"
             }]
     )
     def test_validation_list_with_date(self):
-        """Used to test date validations."""
+        """
+        Used to test date validations using IN_BETWEEN validator.
+        This means that expiration_date must be grater or equal than "2020-10-19 10:20:30",
+        or less or equal than "2020-12-04 10:20:30".
+        """
         fake_object_target_enroll = CourseEnrollment.objects.create()  # pylint: disable=no-member
+
         Tag.objects.create_tag(
             tag_value="example by eduNEXT",
             tag_type="example_tag_3",
             target_object=fake_object_target_enroll,
             owner_object=self.fake_owner_object,
-            expiration_date=datetime.date(2020, 10, 19),
-            activation_date=datetime.date(2020, 6, 16),
+            expiration_date=datetime.datetime(2020, 10, 19, 10, 20, 30),
+            activation_date=datetime.datetime(2020, 6, 16, 10, 20, 30),
+        )
+
+    @override_settings(
+        EOX_TAGGING_DEFINITIONS=[
+            {
+                "tag_type": "example_tag_3",
+                "validate_tag_value": {"regex": r".*eduNEXT$"},
+                "validate_target_object": "CourseEnrollment",
+                "validate_expiration_date": {"in": ["2020-12-04 10:20:30", "2020-10-19 10:20:30"]},
+                "validate_activation_date": "2020-06-16 10:20:30"
+            }]
+    )
+    def test_validation_list_with_using_in(self):
+        """
+        Used to test date validations using IN validator. This means that expiration_date bust be equal to
+        "2020-12-04 10:20:30" or equal to "2020-10-19 10:20:30".
+        """
+        fake_object_target_enroll = CourseEnrollment.objects.create()  # pylint: disable=no-member
+
+        Tag.objects.create_tag(
```

Comment:
could you verify that the tag was created in an assert section ? here the method doesn't fail however if someone add a broad exception  in the method  create_tag, this test will never fail

Reply:
Yes, you're right. I will check for the id of the object to see if it was created

Reply:
done!